staking/programs/integrity-pool/Cargo.toml

```diff
@@ -21,12 +21,4 @@ publisher-caps = {path = "../publisher-caps", features = ["no-entrypoint"]}
 
 
 [dev-dependencies]
-litesvm = "0.1.0"
-solana-program = { workspace = true }
 solana-sdk = { workspace = true }
```

Comment:
do we still need this?

Reply:
its used in the tests
```
 assert!(
            solana_sdk::borsh0_10::get_packed_len::<SlashEvent>()
                + SlashEvent::discriminator().len()
                == SlashEvent::LEN
        );
```

Reply:
I can replace it with `anchor_lang::solana_program::borsh0_10`

Reply:
fixed